File: katas/8 kyu/calculate-average/solution.js
// calculate-average
/* ------------------------------- MY SOLUTION ------------------------------ */
function findAverage(array) {
  return array !== 'undefined' ? array.reduce((x,y) => x+y ,0)/array.length : 0
}
/* ---------------------------------- TEST ---------------------------------- */
const chai = require("chai");
const assert = chai.assert;
chai.config.truncateThreshold=0;

describe("Basic tests", () => {
  it("Testing for fixed tests", () => {
    assert.strictEqual(findAverage([1,1,1]), 1);
    assert.strictEqual(findAverage([1,2,3]), 2);
    assert.strictEqual(findAverage([1,2,3,4]), 2.5);
  });
});  

File: katas/8 kyu/calculate-average/solution.py
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
def find_average(numbers):
    return sum(numbers)/len(numbers)
# ----------------------------------- TEST ----------------------------------- #
@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(find_average([1, 2, 3]), 2)